src/QsCompiler/TestTargets/Simulation/Example/Example.csproj

```diff
@@ -8,6 +8,8 @@
     <IncludeQsharpCorePackages>false</IncludeQsharpCorePackages>
     <CsharpGeneration>false</CsharpGeneration> <!--we provide our own C# generation for the sake of also generating C# for all references-->
     <NoWarn>0219</NoWarn> <!-- suppress C# warning for unused variables -->
+    <Nullable>enable</Nullable>
```

Comment:
Since this is a Q# project, does this then apply the nullable references feature to the generated C#?

Reply:
Hmm, I didn't think about that, but it looks like it doesn't apply to the generated code (it doesn't compile if I manually add `#nullable enable` to one of the generated files). I'm not sure why.

Reply:
I removed `<Nullable>enable</Nullable>` from the Q# projects since it seems to not be doing anything, and I don't think the generated code works with nullable enabled.